bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -61,6 +83,10 @@
 		<label>Control Setpoint</label>
 		<description>Central heating water setpoint set at boiler</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Setpoint</tag>
```

Comment:
While it is a setpoint, it cannot be controlled by openHAB, the channel si read-only. How do we handle such case?
Also applies to some other channels

Reply:
I don’t know. There are some set point channels where the value cannot be set by OH (but is set elsewhere). And there are some (set point override) channels there the value CAN be set by OH. In my sitemap I display both set points plus the actual value on a single chart so I can see what actual set point was applied, and indeed what actually happened to the controlled value.

Reply:
^
but this has nothing to do with tagging..

Reply:
<img width="1133" alt="image" src="https://github.com/user-attachments/assets/5122a9f3-d062-4911-a887-ea4ab6677340" />

Reply:
> ^ but this has nothing to do with tagging..

I think it has everything to do with the tag. As the tag gives usefull information. 

In general: What is the meaning / purpose of the tag ? I was in the impression that Status is for read only and Controllable is for a editable channels. The tags are from the POV of openHAB. 

This specific channel contains a value that is not changable by the user _anywhere_. The value is 'negotiated' between the thermostat and boiler, taking many parameters into account. Once can set the min and max, but that is a different channel.

Reply:
I can only add that this seems like nothing new, same example here for a channel type used e.g. for an oven where it's not possible to change the target temperature through the API:

https://github.com/openhab/openhab-addons/blob/7c50a0a5323f0ab8e86c70f0aff52fbf638c0642/bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml#L159-L169

I'm not sure about the purpose of it, but it seems correct and shouldn't do any harm?

In general it would be nice with a tuturial about how tags are actually being used and how one can benefit from them. Maybe it already exists somewhere?

Reply:
The graph is a good example. By having both `Measurement`/`Temperature` and `Setpoint`/`Temperature` for the same Thing (or channel group within the Thing), it might be possible for a widget, rule or something to pick up the correlation between the two and present that in some nice way?

Reply:
Once those usages are more clear, we can also adapt these. As there are similar channels tagged like this, i can merge it as is. But to me it still seems weird to have it tagged like this.